health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/configs/UICustomizations.js

```
@@ -15,85 +16,193 @@ const businessServiceMap = {};
-          };
-          const switchText = isActive ? "Active" : "Inactive";
-          return (
-            <>
+        case "CHECKLIST_ROLE":
+          let str = row?.data?.role;
+          if (!str.startsWith("ACCESSCONTROL_ROLES_ROLES_")) {
+            str = "ACCESSCONTROL_ROLES_ROLES_" + str;
+          }
+          return t(str);
+        case "CHECKLIST_TYPE":
+          let str1 = row?.data?.checklistType;
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)